src/extension.ts

```diff
@@ -533,7 +533,7 @@ export function createReplTerminal(context: ExtensionContext): void {
                 'log.txt',
               ),
             )}`,
-            iconPath: new ThemeIcon('compass'),
+            iconPath: new ThemeIcon('repl'),
```

Comment:
Why changing the icon?  The idea was to have an icon that is more unique to Legend than a generic icon that would be hard to associate with the product.